server/util/redact/redact.go

```diff
@@ -67,7 +67,20 @@ func stripURLSecretsFromCmdLine(tokens []string) []string {
 		}
 		tokens[i] = stripURLSecrets(token)
 	}
-	return tokens
+}
+
+func stripRemoteHeadersFromCmdLine(tokens []string) {
+	for i, token := range tokens {
+		// Prevent flag name from being included in redaction pattern.
+		if strings.HasPrefix(token, "--remote_header=") {
+			tokens[i] = "--remote_header=<REDACTED>"
```

Comment:
is it possible to just strip x-buildbuddy-platform.container-registry-username=_json_key --remote_header=x-buildbuddy-platform.container-registry-password? Currently, x-buildbuddy-api-key is already stripped. Some of the fields in remote_header are helpful in debugging. e.g. x-buildbuddy-platform.workload-isolation-type.

Reply:
I did it this way to be consistent with the structuredCommandLine redaction -- I can see the value in having an allowed list of remote headers to keep, though. Can chat more offline